docs/tutorials/02_vqe_advanced_options.ipynb

```diff
@@ -0,0 +1,535 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pylab.rcParams[\"figure.figsize\"] = (12, 8)\n",
+    "for i, optimizer in enumerate(optimizers):\n",
+    "    pylab.plot(\n",
+    "        converge_counts[i],\n",
+    "        abs(ref_value - converge_vals[i]),\n",
+    "        label=type(optimizer).__name__,\n",
+    "    )\n",
+    "pylab.xlabel(\"Eval count\")\n",
+    "pylab.ylabel(\"Energy difference from solution reference value\")\n",
+    "pylab.title(\"Energy convergence for various optimizers\")\n",
+    "pylab.yscale(\"log\")\n",
+    "pylab.legend(loc=\"upper right\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Gradients\n",
+    "\n",
+    "In Qiskit's variational algorithms, if the provided optimizer uses a gradient-based technique, the default gradient method will be finite differences. However, these classes include an option to pass custom gradients via the `gradient` parameter, which can be any of the provided methods within Qiskit's [gradient](https://qiskit.org/documentation/stubs/qiskit.algorithms.gradients.html) framework, which fully supports the use of primitives. This section shows how to use custom gradients in the VQE workflow.\n",
+    "\n",
+    "The first step is to initialize both the corresponding primitive and primitive gradient:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_algorithms.gradients import FiniteDiffEstimatorGradient\n",
+    "\n",
+    "estimator = Estimator()\n",
+    "gradient = FiniteDiffEstimatorGradient(estimator, epsilon=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, you can inspect an SLSQP run using the [FiniteDiffEstimatorGradient](https://qiskit.org/documentation/stubs/qiskit.algorithms.gradients.FiniteDiffEstimatorGradient.html) from above:"
```

Comment:
```suggestion
    "Now, you can inspect an SLSQP run using the [FiniteDiffEstimatorGradient](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.gradients.FiniteDiffEstimatorGradient.html) from above:"
```
I noticed this another one that needed updating. (A search would be safest as I mentioned above to doub-check - not sure I'll spot them all as I am reviewing these in the html though I am hovering to check what the links refer to)